scripts/linters/pylint_extensions.py

```diff
@@ -1751,40 +1751,34 @@ class NewlineBelowClassDocstring(checkers.BaseChecker):
     }
 
     def visit_classdef(self, node):
-        """Process a module to ensure that there is a single newline below
-        class docstring.
+        """Visit each class definition in a module.
 
         Args:
-            node: astroid.scoped_nodes.Function. Node to access module content.
+            node: astroid.scoped_nodes.ClassDef. Node for a class definition
+                in the AST.
         """
+        # Check if the given node has docstring.
+        if node.doc is None:
+            return
         line_number = node.fromlineno
-        # Iterate till the end of class definition because some classes has
-        # definition of more than one line. Example:
-        # class ClassName(
-        #       BaseClass):
+        # Iterate till the start of docstring.
         while True:
             line = linecache.getline(node.root().file, line_number).strip()
-            if line.find(b'):') != -1:
+            if line.startswith('"""'):
```

Comment:
Make sure that we only count triple quote as a docstring*

Reply:
@DubeySandeep I don't get this comment of your's can you please explain?

Reply:
A doc string can be like this:

```
class A:
    '''Some Docstring!'''

    pass
```

(and in other different forms right?) So I wasn't you to check whether we are sure that the docstrings are going to be of form `"""`?

Reply:
@DubeySandeep We already have a check for this which only allow docstring starting with` """`

Reply:
Are we sure that this check will run before that check? Also, what will happen if the docsting is written with `'''` will this check will run, if yes then when the `while` loop will terminate?

Reply:
No, but I am thinking of checking all the possible cases, i.e.  `'`, `"`, `'''`, `"""`

Reply:
hmmm.. that can be done! I wanted to check why we removed the earlier `):` check?

Reply:
Because if we used that then the check will fail if there are newlines between the class definition and docstring.